solr/core/src/java/org/apache/solr/update/processor/DistributedZkUpdateProcessor.java

```diff
@@ -325,6 +325,24 @@ protected void doDeleteById(DeleteUpdateCommand cmd) throws IOException {
 
   @Override
   protected void doDistribDeleteById(DeleteUpdateCommand cmd) throws IOException {
+
+    zkCheck();
```

Comment:
This is more of a comment, why is this called zkCheck?   icky name.

Reply:
It verifies that zookeeper (zk) is available.

Reply:
Thanks...   I am tempted to create a JIRA but.....   Will probably just live iwth it.